yogstation/code/modules/mob/living/carbon/human/species_types/preternis/preternis.dm

```diff
@@ -165,9 +165,6 @@ adjust_charge - take a positive or negative value to adjust the charge level
 	if(H.reagents.has_reagent(/datum/reagent/oil))
 		H.adjustFireLoss(-2*REAGENTS_EFFECT_MULTIPLIER,FALSE,FALSE, BODYPART_ANY)
```

Comment:
should probably remove this too, otherwise preterni will double heal

Reply:
I made oil require the PROCESS_SYNTHETIC reagent tag to process which preternis don't have, so they keep that in order to still be able to heal from it